src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -232,7 +232,9 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 				let cells = [...new Set(oldDecorationsRange.map(item => item.cell))].filter(c => c.cellType === 'markdown');
 				cells.forEach(cell => {
 					let cellOldDecorations = oldDecorationsRange.filter(r => r.cell === cell);
-					let cellEditor = this.cellEditors.find(c => c.cellGuid() === cell.cellGuid);
+					// In split mode we have code and text cells with the same Guid,
+					// we need to find the text component editor => c.hasEditor() filters that.
```

Comment:
Update comment - and it would be good for all these checks to have a comment explaining why you're even doing the `getEditor()` check